psalm-baseline.xml

```diff
@@ -0,0 +1,26 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<files psalm-version="4.7.3@38c452ae584467e939d55377aaf83b5a26f19dd1">
+  <file src="src/Controller/Api/GalleryController.php">
+    <MissingDependency occurrences="3">
```

Comment:
Shouldn't we add the dependency ?

Reply:
We have the Rest bundle, but it is complaining because the rest bundle requires sensio framework extra bundle and it does not include it. I don't know how, but on notification bundle it works and doesn't complain with this error.

Reply:
Added